com.deflemask.DefleMask.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.1
+
+---
+
+app-id: "com.deflemask.DefleMask"
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
```

Comment:
Update runtime to 19.08

Reply:
There is no `19.08` runtime on `i386`, sorry.